crates/oxc_linter/src/lib.rs

```diff
@@ -90,6 +115,44 @@ impl Linter {
             }
         }
 
+        let adapter = LintAdapter { semantic: semantic.clone() };
+        let la = Arc::from(adapter);
```

Comment:
I'd merge these two:
```suggestion
        let adapter = Arc::from(LintAdapter { semantic: semantic.clone() });
```
then use `adapter` instead of `la`.

Reply:
Done, thanks!